modules/clearList.js

```diff
@@ -0,0 +1,22 @@
+import updateBookData from './updateBookData.js';
+import displayLists from './displayLists.js';
+
+export default function clearList(taskList) {
```

Comment:
- [ ] Kindly check my comment on the use of `es6` on the addToList file which is very detailed 👍